docs/source/user_guide/pipelines.ipynb

```diff
@@ -372,12 +366,45 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
+    "pipeline = MulticlassClassificationPipeline(['Imputer', 'Random Forest Classifier'])\n",
+    "pipeline.fit(X, y)\n",
     "pipeline.input_feature_names"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Saving and Loading Pipelines\n",
+    "\n",
+    "You can save and load trained or untrained pipeline instances using the Python [pickle](https://docs.python.org/3/library/pickle.html#:~:text=%E2%80%9CPickling%E2%80%9D%20is%20the%20process%20whereby,back%20into%20an%20object%20hierarchy.) format, like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pickle\n",
+    "\n",
+    "pipeline_to_pickle = MulticlassClassificationPipeline(['Imputer', 'Random Forest Classifier'])\n",
+    "\n",
+    "with open(\"pipeline.pkl\", 'wb') as f:\n",
+    "    pickle.dump(pipeline_to_pickle, f)\n",
+    "\n",
+    "pickled_pipeline = None\n",
+    "with open('pipeline.pkl', 'rb') as f:\n",
+    "    pickled_pipeline = pickle.load(f)\n",
+    "\n",
+    "pickled_pipeline.fit(X, y)"
```

Comment:
This is great. I would add
```python
assert pickled_pipeline == pipeline_to_pickle
```
before the call to `fit`, to demonstrate the two pipeline instances are equivalent.